### ISSUE TITLE
TechDraw: Location balloon annotation

### ISSUE THREAD

**Issue by Pendelf** [FreeCAD/FreeCAD#18526]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Problem description

When using balloon annotation, the location of the label is created automatically and as a rule, it has to be moved manually in an inconvenient place. When creating dimensions, let's say an area, first we choose what to measure, then where to place the dimension. Why not do it with the balloon annotation tool

https://github.com/user-attachments/assets/13591bfd-b004-4d43-aa57-f2e0ceae88df

### Full version info

```shell
OS: Windows 11 build 26100
Architecture: x86_64
Version: 1.1.0dev.39646 (Git) Conda
Build type: Release
Branch: main
Hash: b25b83c1a58bf0ae95cbd502d0ccae27e8127cad
Python 3.11.11, Qt 5.15.15, Coin 4.0.3, Vtk 9.3.0, OCC 7.8.1
Locale: Russian/Russia (ru_RU)
Stylesheet/Theme/QtStyle: FreeCAD Dark.qss/FreeCAD Dark/Fusion
Installed mods: 
  * Curves 0.6.52
  * fasteners 0.5.33
  * PieMenu 1.10.2
  * sheetmetal 0.6.2
```

### Subproject(s) affected?

TechDraw

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by pierreporte** [comment]
Yes, it would be better if the annotation feature worked with two clicks, first where it points to, then where the balloon is.
